app/controllers/automated_tests_controller.rb

```diff
@@ -94,7 +94,8 @@ def populate_autotest_manager
       else
         file_keys << file
         { key: file, size: 1,
-          url: download_file_assignment_automated_tests_url(assignment_id: assignment.id, file_name: file) }
+          url: download_file_assignment_automated_tests_url(assignment_id: assignment.id,
+                                                            file_name: File.basename(file)) }
```

Comment:
I don't think this works, `download_files` joins the filename to the top-level autotest files directory (https://github.com/MarkUsProject/Markus/blob/master/app/controllers/automated_tests_controller.rb#L120). I tried it out and downloaded a text file containing a "missing file" message.

Reply:
yup, I added the File.basename in the wrong spot